packages/Maleo.js/src/build/webpack/webpack.ts

```diff
@@ -526,29 +531,16 @@ export const getDefaultOutput = (
 /**
  * Load User Config with file name USER_CUSTOM_CONFIG (maleo.config.js)
  */
-export const loadUserConfig = (dir: string, quiet?: boolean): CustomConfig => {
+export const loadUserConfig = (dir: string): CustomConfig => {
   const cwd: string = path.resolve(dir);
   const userConfigPath: string = path.resolve(cwd, USER_CUSTOM_CONFIG);
-  try {
-    const userConfig = requireRuntime(userConfigPath);
-
-    if (userConfig !== undefined) {
-      // tslint:disable-next-line:no-unused-expression quotemark
-      !quiet && console.log("[Webpack] Using user's config");
-      return {
+  const userConfig = requireFile(userConfigPath);
+  return userConfig
```

Comment:
I think we still need `try` `catch` here. to prevent error if user returns unexpected config type